runtime/runenv_test.go

```diff
@@ -1,10 +1,22 @@
 package runtime
 
 import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
 	"reflect"
+	"strings"
 	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
 )
 
+func init() {
+	_ = os.Setenv("INFLUXDB_URL", "http://localhost:9999")
```

Comment:
Done here: https://github.com/testground/testground/pull/934

I ended up picking `INFLUXDB_ADDR` as a name because the influxdb v1 client refers to the URL as an "Addr", which is wrong because in Go an addr has a different connotation.

I'm putting forth a tiny PR to rename this back to `INFLUXDB_URL`, which is more correct.

And setting this env var is not needed in tests, because we inject a test client anyway.